### PR TITLE
search as component, add to listings page

### PR DESCRIPTION
**Pull request by shapesandobjects**

### CODE REVIEW

File: src/containers/PharmacyList/PharmacyList.tsx [hunk 1]
```diff
@@ -190,9 +191,14 @@ export function PharmacyList(props: Props) {
 
   return (
     <>
+      <section aria-label="search" style={{ marginBottom: "2rem" }}>
```
Comment (chrisahardie) [inline src/containers/PharmacyList/PharmacyList.tsx:194]: Very minor, but there is an inconsistency in that this `section` element isn't in the `Home` component. If it should always be included, it should probably be in the `SearchCard` component.

File: src/components/SearchCard/SearchCard.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+  const invalidPostalCodeMessage = shouldShowInvalidPostal
+    ? t("invalidpostal")
+    : undefined;
+
+  return (
+    <>
+      <Card>
+        <Card.Section>
+          <Form onSubmit={handleSubmit}>
```
Comment (chrisahardie) [inline src/components/SearchCard/SearchCard.tsx:29]: Hmm, there is a `role="search"` attribute value we could apply to search forms as an assistive cue, but it looks like the `Form` component doesn't support aria props... The `Form` props should probably extend from `IntrinsicAttributes` so as to support attributes assignable to HTML elements.

**Review by chrisahardie — DISMISSED** [review]

**Review by ericherscovich — CHANGES_REQUESTED** [review]
When there are no pharmacies available, the search bar doesn't show up. Look K2T 0E5 which has no availabilities and you'll see this.

Edit: Fixed it.

**Review by ericherscovich — APPROVED** [review]

**Comment by ericherscovich** [comment]
Added the 3 digit postal regex to this. Closes #84

**Review by ericherscovich — APPROVED** [review]

**Pull request merged**